crates/bevy_ecs/src/schedule/system_descriptor.rs

```diff
@@ -13,8 +13,9 @@ use crate::{
 /// * At end, accepts exclusive systems; runs after parallel systems' command buffers have
 /// been applied.
 ///
-/// All systems can have a label attached to them; other systems in the same group can then specify
-/// that they have to run before or after the system with that label using the `before` and `after` methods.
+/// All systems can have one or several labels attached to them; other systems in the same group
```

Comment:
I think the overall sentence structure needs work? This is a bit hard to read and leaves the reader confused about whether systems can have 0 labels.

"You can attach any number of labels to any particular system, and labels can be reused freely on other systems."

Reply:
Done in e00f6529e22488d39f95d647f1f7cc8ff6e37548.

> This is a bit hard to read and leaves the reader confused about whether systems can have 0 labels.

"Systems ***can*** have one or more labels attached to them ..." seems unambiguous enough to me. "... before or after system***s*** with that label ..." implies that the labels don't have to be unique. Users reading this don't come from a place where labels had to be unique, because 0.4 didn't have labels at all - and as far as documentation is concerned, 0.5 exists in a vacuum regardless.

Reply:
Yeah the new wording works for me. Feel free to follow up with proposed improvements in another pr :)